tools/packaging/promtail-minimal-config.yaml

```diff
@@ -0,0 +1,23 @@
+# This minimal config scrape only single log file.
+# Primarily used in rpm/deb packaging where promtail service can be started during system init process.
+# And too much scraping during init process can overload the complete system.
+# https://github.com/grafana/loki/issues/11398
+
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: http://localhost:3100/loki/api/v1/push
+
+scrape_configs:
+- job_name: system
+  static_configs:
+  - targets:
+      - localhost
+    labels:
+      job: varlogs
+      __path__: /var/log/messages
```

Comment:
Can we add a comment that this section should be changed by the user after installation?

Reply:
just a hint :-) - there is now more failing in the build-pipeline - https://drone.grafana.net/grafana/loki/32891/25/8

Reply:
"Release stage" on drone pipeline seems flaky.

Reply:
I think found the issue. The packaging test pipeline generate test log and writes to `/var/log/test.log` and checks if its scraped fine, By using minimal config we scrape only `/var/log/messages` and doesn't include `/var/log/test.log`.

I changed the filename from `/var/log/test.log` -> `/var/log/messages` instead in the scripts so test logs are properly scraped.

I think that should fix it.

Reply:
It did.